### PR TITLE
fix: faulty avatars for guests and deleted users

### LINKED ISSUE #13716: Trim whitespaces in guest names
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Have a guest with a whitespace before the name " Marcel"

### Actual behaviour
Sidebar is fine, but callview fails:

Image

## Talk app

**Talk app version:** 20.0.1

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Fix #13716

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | image
image | image
image | image
image| image

🏚️ Before | 🏡 After
-- | --
image | image

### 🚧 Tasks

- [ ] Follow-up: do not pass 'Guest' to AvatarWrapper

### 🏁 Checklist

- [x] 🌏 Tested with different browsers / clients:
  - [x] Chromium (Chrome / Edge / Opera / Brave)
  - [ ] Firefox
  - [ ] Safari
  - [ ] Talk Desktop
  - [x] Not risky to browser differences / client
- [x] ⛑️ Tests are included or not possible

### CODE REVIEW

File: src/utils/getDisplayName.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		return t('spreed', 'Guest')
+	}
+
+	if (source === ATTENDEE.ACTOR_TYPE.DELETED_USERS) {
+		return t('spreed', 'Deleted user')
+	}
+
+	// Fallback to 'Deleted user': should not happen, but can not be empty either
+	return t('spreed', 'Deleted user')
+}
```
Comment (DorraJaouad) [inline src/utils/getDisplayName.ts:24]: can be merged into one return

**Review by DorraJaouad — APPROVED** [review]
Tested

File: src/utils/getDisplayName.ts (hunk 1 above)
Comment (Antreesy) [reply src/utils/getDisplayName.ts:24]: Logic is different in two cases, but for the sake of reducing code keeping only fallback return with comment in code source

**Pull request merged**